provider/netcup.go

```diff
@@ -209,35 +227,39 @@ func (p *NetcupProvider) ApplyChanges(ctx context.Context, changes *plan.Changes
 		}
 
 		// If not in dry run, apply changes
-		_, err = p.session.UpdateDnsRecords(zoneName, change.UpdateOld)
-		if err != nil {
-			return err
-		}
+		// we don't need to delete old records, we can just update
+		//p.logger.DebugContext(ctx, fmt.Sprintf("UpdateOld"), "zone", zoneName, "records", change.UpdateOld)
+		//_, err = p.session.UpdateDnsRecords(zoneName, change.UpdateOld)
```

Comment:
Do you agree that this is not needed? Or are there cases where the record must be deleted before applying an update?

Reply:
Thanks for working on additional tests!